frontend/src/model/beta/releases/ModelReleaseDisplay.tsx

```diff
@@ -108,37 +115,39 @@ export default function ModelReleaseDisplay({
             <Box>{(release.files.length > 0 || release.images.length > 0) && <Divider />}</Box>
             <Stack spacing={2}>
               {release.files.length > 0 && (
-                <Box>
+                <>
                   <Typography fontWeight='bold'>Artefacts</Typography>
                   {release.files.map((file) => (
                     <Stack
                       key={file._id}
                       direction={{ sm: 'row', xs: 'column' }}
                       justifyContent='space-between'
                       alignItems='center'
-                      spacing={2}
+                      spacing={1}
                     >
                       <Link href='/beta'>{file.name}</Link>
                       <Typography variant='caption'>{prettyBytes(file.size)}</Typography>
                     </Stack>
-                  ))}{' '}
-                </Box>
+                  ))}
+                </>
               )}
               {release.images.length > 0 && (
-                <Box>
+                <>
                   <Typography fontWeight='bold'>Docker images</Typography>
                   {release.images.map((image) => (
                     <Stack
-                      key={`${image.model}-${image.version}`}
+                      key={`${image.name}-${image.tag}`}
```

Comment:
In theory I think we could have two images for different models, both with the same name and tag. Is that correct?

If so, consider adding the model id to the key also.